Source/WebCore/platform/mediastream/gstreamer/GStreamerAudioCaptureSource.cpp

```diff
@@ -87,7 +87,7 @@ AudioCaptureFactory& GStreamerAudioCaptureSource::factory()
 
 GStreamerAudioCaptureSource::GStreamerAudioCaptureSource(GStreamerCaptureDevice device, MediaDeviceHashSalts&& hashSalts)
     : RealtimeMediaSource(device, WTFMove(hashSalts))
-    , m_capturer(makeUnique<GStreamerAudioCapturer>(device))
+    , m_capturer(makeUnique<GStreamerAudioCapturer>(WTFMove(device)))
```

Comment:
Did you by any change forget to add the `&&` do the device parameter?

Reply:
Yes